examples/log/index.html

```diff
@@ -0,0 +1,295 @@
+        button.addEventListener('click', () => UI.log.error('An error message'))
+        document.querySelector('#demo-log-error').appendChild(button)
+      })
+    </script>
+    <pre id="viewSource-log-error"></pre>
+    <div id="demo-log-error"></div>
+</article>
+
+<article id="log-success">
+    <h2>
+        <a href="#log-success">
+            <code>log.success</code>: A success message
+        </a>
+    </h2>
+    <script id="script-log-success">
+      window.addEventListener('DOMContentLoaded', () => {
+        const button = document.createElement('button')
+        button.innerText = 'Add a success message'
+        button.addEventListener('click', () => UI.log.success('A success message'))
+        document.querySelector('#demo-log-success').appendChild(button)
+      })
+    </script>
+    <pre id="viewSource-log-success"></pre>
+    <div id="demo-log-success"></div>
+</article>
+
+<article id="log-alert">
+    <h2>
+        <a href="#log-alert">
+            <code>log.alert</code>: An alert message
+        </a>
+    </h2>
+    <p>
+        This example uses alert to notify user of a message. It will not log in the status area unless
+        <code>window.alert</code> is not available.
+    </p>
+    <script id="script-log-alert">
+      window.addEventListener('DOMContentLoaded', () => {
+        const button = document.createElement('button')
+        button.innerText = 'Trigger an alert message'
+        button.addEventListener('click', () => UI.log.alert('An alert message'))
+        document.querySelector('#demo-log-alert').appendChild(button)
+      })
+    </script>
+    <pre id="viewSource-log-alert"></pre>
+    <div id="demo-log-alert"></div>
+</article>
+
+<article id="log-clear">
+    <h2>
+        <a href="#log-clear">
+            <code>log.clear</code>: Clear messages
+        </a>
+    </h2>
+    <p>
+        Using this method you can clear the existing messages in the status area.
+    </p>
+    <script id="script-log-clear">
+      window.addEventListener('DOMContentLoaded', () => {
+        const button = document.createElement('button')
+        button.innerText = 'Clear status area'
+        button.addEventListener('click', () => UI.log.clear())
+        document.querySelector('#demo-log-clear').appendChild(button)
+      })
+    </script>
+    <pre id="viewSource-log-clear"></pre>
+    <div id="demo-log-clear"></div>
+</article>
+
+<article id="log-setLevel">
+    <h2>
+        <a href="#log-setLevel">
+            <code>log.setLevel</code>: Configure which messages that are to be shown
+        </a>
+    </h2>
+    <p>
+        You can fine-tune which messages you want shown by adding together the corresponding numbers for each level:
+    </p>
+    <ul>
+        <li>Error: 1</li>
+        <li>Warning: 2</li>
+        <li>Message: 4</li>
+        <li>Success: 8</li>
+        <li>Info: 16</li>
+        <li>Debug: 32</li>
+    </ul>
+    <p>
+        Setting level to 64 will show all messages.
```

Comment:
63, not 64. You could also clarify "It's a bitmask, so setting level to 63 (i.e. 1+2+4+8+16+32) will show all messages."

Reply:
Yeah, was wondering whether or not to mention the bitmask thing, as some might be confused by it. But I'll phrase it in such a way that is useful for those that understand and not necessary to understand for those who don't.